server/posts/tests/models_tests.py

```diff
@@ -78,3 +78,7 @@ def test_overriding_defaults(self):
     def test_get_comments_page_url(self):
         post = self.create_post()
         self.assertRegex(post.get_comments_page_url(), r'^http.+/author/.+/posts/.+/comments')
+    
+    def test_get_id_url(self):
+        post = self.create_post()
+        self.assertRegex(post.get_comments_page_url(), r'^http.+/author/.+/posts/.+')
```

Comment:
Try using reverse function and you can create a global static variable at the top of the file to clean up the code
```suggestion
        self.assertRegex(post.get_comments_page_url(), reverse('author:posts'))
```

Reply:
We don't have an author:post url. And even if we have it, it wouldn't be clear what we are testing it against.
This is to explicitly test whether we are getting the URL in a certain format, and not if we are getting the same URL back. Let me know what you think!

Reply:
Update: this returns the frontend URL (e.g.  `https://www.konnection.com/author/<user_uuid>/post/<post_id>`) and reverse() returns the backend URL (with `service/` between `www.konnection.com/` and `author`). But I do need to fix this line because it is testing against the wrong URL. Will update the regex.